api_tests/tests/bitcoin_ethereum.ts

```diff
@@ -1,67 +1,11 @@
 /**
```

Comment:
what was the reason for moving it out of the `e2e` folder? I liked the folder structure :)

Reply:
It just added noise IMO and our new test_environment does not longer care about dry vs e2e tests. I just runs tests and starts things if the tests require that.

I would expect that the structure of these tests gets a lot clearer with https://github.com/comit-network/comit-rs/issues/2193.